scalyr_agent/builtin_monitors/linux_process_metrics.py

```diff
@@ -243,8 +255,12 @@ def is_cumulative(self):
             "app.uptime",
             "app.disk.bytes",
             "app.disk.requests",
+            "app.io.wait",
```

Comment:
I believe the major page fault count is cumulative as well, correct?  It is the number since the process began.

Reply:
Correct, pushed a fix - 43038a5c3a50827741a21501d98f100ed72a7c1b.